style.css

```diff
@@ -0,0 +1,9 @@
+body {
+  width: 80%;
+  font-family: 'IBM Plex Serif', serif;
```

Comment:
At this point, had you already pulled this font in w/ Google Fonts or similar? This seems like a font that's not available on every machine.